doc/conf.py

```diff
@@ -136,17 +136,17 @@
     "filename_pattern": r"\.py",
     # Remove the "Download all examples" button from the top level gallery
     "download_all_examples": False,
-    # Sort gallery example by file name instead of number of lines (default)
+    # Sort gallery examples by file name instead of number of lines (default)
     "within_subsection_order": ExampleTitleSortKey,
-    # directory where function granular galleries are stored
+    # Directory where function granular galleries are stored
     "backreferences_dir": "api/generated/backreferences",
-    # Modules for which function level galleries are created.  In
-    # this case sphinx_gallery and numpy in a tuple of strings.
+    # Modules for which function level galleries are created. In
+    # this case sphinx_gallery and numpy are a tuple of strings.
```

Comment:
```suggestion
    # Modules for which function level galleries are created.
```

Reply:
Reformulated in commit https://github.com/GenericMappingTools/pygmt/pull/3573/commits/845aa96af7229c5911559d3008d6ef0744245dbd.